### DISCUSSION TITLE
LaTeX not rendered properly anymore

### DISCUSSION THREAD

**Discussion by klimanek** [sympy/sympy#26959] [General]
What has happened in past week or so that math typesetting got so messy? Fonts don't look like TeX family, formulas are tiny and not readable well. When I was opening a new issue last week and used math they looked just fine. The issue seems to be GitHub-wide. 

Example:
![tex_not_rendered](https://github.com/user-attachments/assets/c5410a58-bcdc-46d7-bec2-5ade11ebb047)

Should look like this:
![tex_rendered_ok](https://github.com/user-attachments/assets/8a652778-4124-4b76-9509-fc27cca49624)

**Comment by asmeurer** [comment]
I think it means mathjax isn't loading the fonts correctly. It isn't related to sympy.

**Reply by klimanek** [reply]
Yes, I am aware of that. As a temp workaround, I have installed the [MathJax for Github](https://chromewebstore.google.com/detail/mathjax-plugin-for-github/ioemnmodlmafdkllaclgeombjnmnbima) extension and it seems to work. Now all the math in issues and comments is readable again.

**Comment by harshisageek** [comment]
hyy everyone i am new here and wish to contribute to this project can someone guide me
